packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -61,14 +72,19 @@ class CupertinoCheckbox extends StatefulWidget {
     required this.value,
     this.tristate = false,
     required this.onChanged,
+    this.mouseCursor,
     this.activeColor,
     this.inactiveColor,
+    this.fillColor,
     this.checkColor,
     this.focusColor,
     this.focusNode,
     this.autofocus = false,
     this.side,
     this.shape,
+    this.shadowColor,
+    this.isError = false,
+    this.semanticLabel,
```

Comment:
overlayColor and splashRadius are missing from CupertinoCheckbox but are irrelevant since there is no Material?

Reply:
Exactly.